Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/BuildTelescopeDetector.hpp

```diff
@@ -34,7 +34,8 @@ enum class TelescopeSurfaceType {
 ///
 /// @param gctx is the detector element dependent geometry context
 /// @param detectorStore is the store for the detector element
-/// @param positions is the offset w of different layers in the longitudinal
+/// @param positions is the offset of different layers in the longitudinal
+/// @param stereos is the stereo angle of different layers around the longitudinal
```

Comment:
```suggestion
/// @param stereos is the stereo angle (i.e., ...) of different layers around the longitudinal direction
```

Reply:
Could you maybe give a short explanation of what a stereo angle is? I guess it is something like the polar angle phi?

Reply:
To me the descriptions seems relative clear: rotation around the longitudinal direction. I'll try to clarify it a bit more, anyway.